vm/devices/user_driver/src/vfio.rs

```diff
@@ -55,13 +57,13 @@ pub struct VfioDevice {
     #[inspect(skip)]
     device: Arc<vfio_sys::Device>,
     #[inspect(skip)]
-    dma_buffer: Arc<dyn VfioDmaBuffer>,
-    #[inspect(skip)]
     msix_info: IrqInfo,
     #[inspect(skip)]
     driver_source: VmTaskDriverSource,
     #[inspect(iter_by_index)]
     interrupts: Vec<Option<InterruptState>>,
+    #[inspect(skip)]
+    dma_client: Arc<dyn DmaClient>,
```

Comment:
I think we should instead require `DmaClient` implement Inspect so that it's inspectable. You can imagine we'd might want to report stuff such as dma policy, outstanding dma allocations, etc.